stdlib/http/src/main/ballerina/http/client_endpoint.bal

```diff
@@ -205,6 +207,8 @@ public type AuthConfig record {
     string tokenUrl;
     string clientId;
     string clientSecret;
+    ClientAuthentication clientAuthentication = BASIC_AUTH_HEADER;
+    string tokenScope;
```

Comment:
Shall we rename `tokenScope` into `scope` in order to match with specification?
https://tools.ietf.org/html/rfc6749#page-47

Reply:
Unfortunately `scope` is a keyword in ballerina :D

Reply:
Any other better suggestion instead of `tokenScope`?

Reply:
Since it is a keyword, can we use like `^"scope"`?

Reply:
Yes we can. Will do the change